examples/singleElement_20y_inp/alara_inp_fe_20y

```diff
@@ -0,0 +1,38 @@
+geometry rectangular
+
+volume
+     10    zone_1
+end
+
+mat_loading
+    zone_1    mix_1
+end
+
+mixture mix_1
+    material mat:fe 1 1.0
+end
+
+material_lib fe_matlib.txt
```

Comment:
I'm ok with the introduction of a custom material library, but probably makes more sense for something that isn't just defining a raw element